activiti-api-process-model/src/main/java/org/activiti/api/process/model/payloads/TimerPayload.java

```diff
@@ -0,0 +1,154 @@
+package org.activiti.api.process.model.payloads;
+
```

Comment:
there are double space in the all file

Reply:
Double lines are removed:
https://github.com/Activiti/activiti-api/pull/128/commits/424814493381e36587d96212a6da7ba6ce5e5b16